opentreemap/treemap/templatetags/form_extras.py

```diff
@@ -27,7 +27,7 @@
 # model.udf:field name, can't be done in the grammar as it can't be checked
 # until looked up in the context
 _identifier_regex = re.compile(
-    r"^[a-zA-Z_.\-]+(?:udf\:[\w '\._-]+|[a-zA-Z0-9_\-]+)$")
+    r"^[a-zA-Z_.\-]+(?:udf\:.+|[a-zA-Z0-9_\-]+)$")
```

Comment:
I purposefully didn't do that.  We may need to add more restrictions to the UDF name regex in the future, and I thought it would quickly get out of sync from this regex.  This only really "sanity checking", so there isn't a need to be as restrictive here as we are when creating UDFs.